plonetheme/onegov/resources/filter.html

```diff
@@ -0,0 +1,364 @@
+                  <span class="wrapper"><a title="" href="#">Beh&ouml;rden</a></span>
+                </li>
+              </ul>
+
+            </div>
+            <div class="cell position-12 width-4 searchBoxWrapper">
+              <div id="portal-searchbox">
+
+                <form action="#search" id="livesearch0" name="searchform">
+                  <div class="LSBox">
+                    <input type="text" id="searchGadget" class="searchField inputLabel" placeholder="OneGovBox durchsuchen" />
+                    <input type="submit" value="Suche" class="searchButton" />
+                  </div>
+                </form>
+
+              </div>
+            </div>
+          </div>
+          <div class="row headerDelimiterWrapper">
+            <div class="cell position-0 width-16" id="header-delimiter">&nbsp;</div>
+          </div>
+          <!-- closes #header -->
+        </div>
+
+        <div id="columns" class="clearfix">
+
+          <div class="row">
+            <div class="cell position-0 width-16" id="breadcrumbs-wrapper">
+              <div id="portal-breadcrumbs">
+                <span id="breadcrumbs-you-are-here">Sie sind hier:</span>
+                <span id="breadcrumbs-home">
+                  <a href="http://www.zg.ch">Startseite</a>
+                </span>
+              </div>
+
+              <div id="document-actions">
+                <ul><li id="document-action-print">
+                  <a href="javascript:this.print();" title="Drucken">5</a>
+                </li></ul>
+              </div>
+
+              <div class="visualClear"><!-- --></div>
+
+            </div>
+          </div>
+
+          <div class="row">
+            <div class="cell position-0 width-16" id="banner-image"></div>
+          </div>
+
+          <div class="row">
+
+            <div class="cell position-0 width-16" id="column-content">
+
+
+              <h1> Option 1 - filter directly</h1>
+              <div id="filter-form">
+                <div>
+                  <div class="LSBox filter">
+                        <input name="SearchableText" type="text" size="18" title="Search" class="searchField inputLabel" placeholder="Search" id="searchGadget" autocomplete="off">
+                  </div>
+                  <div class="filter">
+                    <span>Type</span>
+                    <ul>
+                      <li><a>Option 1</a></li>
+                      <li><a>Option 2</a></li>
+                      <li><a>Option 3</a></li>
+                    </ul>
+                  </div>
+                  <div class="filter">
+                    <span>Topic</span>
+                    <ul>
+                      <li><a>Option 1</a></li>
+                      <li><a>Option 2</a></li>
+                      <li><a>Option 3</a></li>
+                    </ul>
+                  </div>
+                  <div class="filter">
+                    <span>Last change</span>
+                    <ul style="display: block;">
```

Comment:
For the mockup... if your opening the site you don't have to click around, instead it shows directly the dropdown (check image). display:block ist what the js part als does.